### ISSUE TITLE
1.4.1 and 1.5.0 unusable

### ISSUE THREAD

**Issue by dpiddockcmp** [terraform-aws-modules/terraform-aws-vpc#33]
Hi,

https://github.com/terraform-aws-modules/terraform-aws-vpc/commit/b684241e0c25eed03a96eaed4f2d3bbc522748e7 removed a bunch of variables that it shouldn't have. Releases since (1.4.1 and 1.5.0) are totally unusable. They break all unversioned inclusions of the module:

```
Error getting plugins: module root: 
  module banksites-vpc.root: 8 error(s) occurred:

* resource 'aws_vpc_dhcp_options.this' config: unknown variable referenced: 'dhcp_options_domain_name'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options.this' config: unknown variable referenced: 'dhcp_options_domain_name_servers'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options.this' config: unknown variable referenced: 'dhcp_options_ntp_servers'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options.this' config: unknown variable referenced: 'dhcp_options_netbios_name_servers'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options.this' config: unknown variable referenced: 'dhcp_options_netbios_node_type'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options.this' count: unknown variable referenced: 'enable_dhcp_options'. define it with 'variable' blocks
* resource 'aws_vpc_dhcp_options_association.this' count: unknown variable referenced: 'enable_dhcp_options'. define it with 'variable' blocks
* resource 'aws_vpn_gateway.this' count: unknown variable referenced: 'enable_vpn_gateway'. define it with 'variable' blocks
```

**Comment by antonbabenko** [comment]
Fixed in v1.5.1 and force-pushed v1.5.0. Human error. My bad.

**Comment by antonbabenko** [comment]
Fixed v1.4.1 also.

@dpiddockcmp Thank you for opening this issue so fast!
